lib/honeydew.ex

```diff
@@ -217,19 +229,32 @@ defmodule Honeydew do
   @doc """
   Creates a supervision spec for workers.
 
-  `queue` is the name of the queue that the workers pull jobs from
-  `module` is the module that the workers in your queue will use, you may also provide init/1 args: {module, args}
+  `queue` is the name of the queue that the workers pull jobs from.
+
+  `module` is the module that the workers in your queue will use, you may also
+  provide `c:Honeydew.Worker.init/1` args with `{module, args}`.
```

Comment:
looks like a typo here

Reply:
Are you talking about the `c:` in front of the function name? That's a callback link. I was thinking of rewriting to:

```
`module` is the module that the workers in your queue will use. You may also
provide `c:Honeydew.Worker.init/1` args with `{module, args}`.
```

Reply:
ah, my bad, i've never seen that before. :)

is that an ex_doc thing?

Reply:
Yep. Check out the bottom paragraph of this auto-linking section in the ex_doc readme: https://github.com/elixir-lang/ex_doc#auto-linking